index.html

```diff
@@ -1107,8 +1107,35 @@
 
 ## Principles for Notifications and Interruptions
 
-<aside class="issue">Connect intrusiveness to [=autonomy=], and write some principles.</aside>
+<aside class="issue">This is relateds to [=autonomy=].</aside>
 
+Notifications and other interruptive UI can be a powerful way to capture attention. 
+Depending on the operating system in use, a notification can eppear outside of the
+browser context (for example, in a general notifications tray) or even cause a device to
+buzz or play an alert tone.
+Like all powerful features, notifications can be misused to and can become an annoyance 
+or even used to manipulate behaviour and thus reduce [=atonomy=].
+
+<div class="practice">
+
+  <span class="practicelab" id="principle-notifications control">A [=user agent=] should help
+    users control notifications and other interruptive UI that can be used to manipulate behavior.</span>
+
+  [=User agents=] should provide UI that allow's their user to audit which web [=sites=] have been granted 
+  permission to display alerts and to revoke these permissions.  
+  [=User agents=] may also apply some quality metric to the initial request for permissions to receive
+  notifications (for example, disallowing sites from requesting permission on first visit).
+  
+  <span class="practicelab" id="principle-notifications-context">Web [=sites=] should use notifications
+    only for information that their users have spedcifically requested.
+  </span>
+  
+  Web [=sites=] should be specific with their users about what kind of information they can expect to 
+  receive, and how notifications can be turned off,  when requesting permission to send interruptive 
+  notifications.  Web [=sites=] should not request permission to send notifications on first visit.
```

Comment:
How about "web sites should not request permission to send notifications when the user is unlikely to have sufficient knowledge to make an informed response"  (Example: you visit a web-based collaborative tool, go through the tutorial and add the other collaborators on your project. You're still on your first visit, and the site asks you "would you like a notification when one of your collaborators makes a change to this project?") A turn-based game might also ask permission on your first visit since you might expect to be notified when it's your turn.

(Similar to the problem of when to request consent for processing: https://github.com/w3ctag/privacy-principles/pull/128#discussion_r807519650 )

Reply:
```suggestion
  notifications.  Web [=sites=] should not request permission to send notifications when the user is unlikely to have sufficient knowledge (e.g. information about what kinds of notifications they are signing up for) to make an informed response. If it's unlikely that such information could have been provided then the [=user agent=] should apply mitigations (for example, warming about potential malicious use of the notifications API).
```

Reply:
wdyt?

Reply:
Looks good to me (I was mainly thinking about the case where someone sends you a link to an article, you click through, and you get prompted to allow notifications before you read the article)

"If it's unlikely that such information could have been provided..."

"If the user is new to the site, or unlikely to be familiar with it..."